go.mod

```diff
@@ -1,6 +1,6 @@
 module google.golang.org/grpc
 
-go 1.19
+go 1.21
```

Comment:
Similar question here.

Reply:
I do have a question about this, I don't think you use any feature of the language that isn't in 1.19, and that change then propagates to my library which, while I personally build binary with 1.22.5, I'd rather not force upgrade yet for users (and I was planning on a minor patch level bump but if my go.mod changes from 1.19 to 1.21 I'd have to make a new minor)

Reply:
nvm
> stats/opentelemetry module pulls in dependency which requires go1.21+
so I guess I would need to track down why _they_ bumped it... possibly unecessarily

Reply:
I think this is the new normal for the Go ecosystem, particularly after the changes made to go.mod in 1.21.  We won't bump this unless we have to, but it will become more and more likely that we'll need to shortly after any new Go version is released.